crates/sui-source-validation-service/README.md

```diff
@@ -0,0 +1,76 @@
+# Sui Source Validation Service
+
+This document describes the Sui Source Validation Service. It is engineering documentation primarily for MystenLabs engineers who may want to build, extend, configure, or understand the service.
```

Comment:
(There's a space in the company name when used in copy, even though the logo with text doesn't include a space).

```suggestion
This document describes the Sui Source Validation Service. It is engineering documentation primarily for Mysten Labs engineers who may want to build, extend, configure, or understand the service.
```

Reply:
thanks, the logo has somehow ingrained the lack of space in my head